jerry-core/parser/js/js-parser.c

```diff
@@ -1439,6 +1441,9 @@ parser_append_breakpoint_info (parser_context_t *context_p, /**< context */
 static ecma_compiled_code_t *
 parser_post_processing (parser_context_t *context_p) /**< context */
 {
+#if defined  JERRY_DEBUGGER || defined  JERRY_ENABLE_LINE_INFO
+  JERRY_DEFINE_CURRENT_CONTEXT ();
+#endif
```

Comment:
Style: I don't think the code base uses `defined  FEATURE_GUARD` style anywhere but `defined (FEATURE_GUARD)` (one space + parentheses). (And even this commit uses the latter style elsewhere.) Plus, `#endif` misses a comment.